File: uri/python/1117.py
soma = 0
conta_notas = 0

while True:
    if conta_notas == 2:
        print('media = {:.2f}'.format(soma/2))
        break
    nota = float(input())
    if nota < 0 or nota > 10:
        print('nota invalida')
        continue
    else:
        soma += nota
        conta_notas += 1

    

File: uri/c/1117.c
#include <stdio.h>

int main() {

    float nota, tmp, media = 0;
    int validas = 1;

    while (validas != 3) {

        scanf("%f", &nota);

        if (nota < 0 || nota > 10) {
            printf("nota invalida\n");
            continue;
        }

        if (validas == 2) {
            media = (tmp + nota) / 2.0;
            printf("media = %.2f\n", media);
        }
        
        tmp = nota;
        validas++;

    }


    return 0;

}